### ISSUE TITLE
crash report

### ISSUE THREAD

**Issue by canyaman** [telepresenceio/telepresence#1063]
### What were you trying to do?

run telepresence --run-shell

### What happened instead?

(please tell us - the traceback is automatically included, see below.
use https://gist.github.com to pass along full telepresence.log)

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '--run-shell']`
Version: `0.101`
Python version: `3.7.3 (default, Mar 27 2019, 09:23:15) 
[Clang 10.0.1 (clang-1001.0.46.3)]`
kubectl version: `Client Version: v1.15.0 // Server Version: v1.11.0+d4cacc0`
oc version: `Client Version: version.Info{Major:"4", Minor:"1+", GitVersion:"v4.1.0+b4261e0", GitCommit:"b4261e07ed", GitTreeState:"clean", BuildDate:"2019-05-18T05:40:34Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"darwin/amd64"} // Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2019-06-20T16:29:27Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}`
OS: `Darwin Cans-MacBook-Pro.local 18.6.0 Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64`

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 136, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 102, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 153, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 100, in get_deployment_json
    )["items"][0]
IndexError: list index out of range

```

Logs:

```
m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1561552111-629418-51999 --image=datawire/telepresence-k8s:0.101 --labels=telepresence=f4a2767d9af744ab80aa427a940157b7 --env=TELEPRESENCE_NAMESERVER=8.8.4.4
   3.0  12 | kubectl run --generator=deploymentconfig/v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
   3.1  12 | deploymentconfig.apps.openshift.io/telepresence-1561552111-629418-51999 created
   3.1 TEL | [12] ran in 0.27 secs.
   3.1 TEL | END SPAN deployment.py:97(create_new_deployment)    0.5s
   3.1 TEL | BEGIN SPAN remote.py:151(get_remote_info)
   3.1 TEL | BEGIN SPAN remote.py:78(get_deployment_json)
   3.1 TEL | [13] Capturing: oc --context xxx-project/192-168-64-36:8443/admin --namespace xxx-project get deployment -o json --selector=telepresence=f4a2767d9af744ab80aa427a940157b7
   3.3 TEL | [13] captured in 0.20 secs.
   3.3 TEL | END SPAN remote.py:78(get_deployment_json)    0.2s

```

**Comment by ark3** [comment]
Sorry about the crash! This is issue #1010.

**Comment by ark3** [comment]
I believe this was fixed by #1404.
